spec/support/shared_examples/bulk_updatable_type.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+GENERATORS = {
+  string: ->(index) { "wat_#{index}" },
+  boolean: ->(index) { index.even? },
+  hstore: ->(index) { { "foo_#{index}" => "bar_#{index}" } },
+  datetime: ->(index) { Time.zone.now.in_time_zone('UTC').round - index.days },
+  jsonb: ->(index) { { "bee_#{index}" => "bizzle_#{index}" } },
+}.freeze
+
+RSpec.shared_examples 'bulk updatable type' do |type|
```

Comment:
How does the different `type` get passed into this test spec? 
From line 81 to 83 in `spec/junk_drawer/rails/bulk_updatable_spec.rb`?

Reply:
yeah, it's the second argument to `it_behaves_like`. Anything that comes after the string description gets passed in here as block arguments.